tests/ecmascript/test-bi-object-is.js

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Object.is()
+ */
+
+/*===
+basic test
+function true false true
+2
+true
+false
+false
+true
+true
+0 0
+1 1
+2 2
+3 3
+4 4
+5 5
+6 6
+7 7
+8 8
+9 9
+10 10
+11 11
+12 12
+===*/
+
+function basicTest() {
+    var pd = Object.getOwnPropertyDescriptor(Object, 'is');
+    print(typeof pd.value, pd.writable, pd.enumerable, pd.configurable);
```

Comment:
When I wrote the ES5 bindings I wrote a testcase to print the property attributes of all built-ins in a separate test case. That leaves the specific test cases to focus on functionality only. But it's rather annoying to maintain because you need to remember the test case even exists, so I've now tried to include e.g. attribute and `.length` tests in the specific tests themselves.

As a sidenote, luckily test cases don't need to be identical in structure, overlap in tests, or even all that well organized: test cases are not part of the build so they don't have footprint issues, and they can just be deleted/rewritten if something becomes too much of an issue.

What matters is regression coverage and ease of writing new tests to maintain that coverage. For example, when new functionality is written, it doesn't really matter if there's some overlap to existing tests. So I try not to be too critical of test cases beyond their coverage :-)